src/widgets/ActivityDetails/model/hooks/useAnswers.ts

```diff
@@ -124,9 +132,10 @@ export const useAnswer = (props: UseAnswerProps) => {
       getGroupInProgressByIds,
       getSubmitId,
       props.activityId,
-      props.appletEncryption,
-      props.appletId,
-      props.appletVersion,
+      props.appletDetails.activityFlows,
+      props.appletDetails.encryption,
+      props.appletDetails.id,
+      props.appletDetails.version,
```

Comment:
Does it make any sense to wrap processAnswers in useCallback?
If so then you can provide just  props.appletDetails as a dependency

Reply:
Done